scripts/main.js

```diff
@@ -0,0 +1,137 @@
+// 1.1
+var nums = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+function getEvenNums1(arr) {
+  if (!Array.isArray(arr)) throw new TypeError;
+
+  let evNums1 = [];
+  for (let i = arr.length - 1; i >= 0; i--) {
+    // Skip if element is not a number.
+    if (typeof arr[i] !== 'number') {
+      continue;
+    }
+
+    if (arr[i] % 2 != 0) continue;
```

Comment:
this if and previous if can be merged
if (typeof arr[i] !== 'number' || arr[i] % 2 != 0)) {
      continue;
}

Reply:
completely agree, did not notice